### PR TITLE
Adds support for $user:$hash:$salt filetype

### PR DESCRIPTION
**Pull request by GrepItAll**
Adds support to solve #373 

Does not add named support for -m 2611, but allows importing of hashfiles in $user:$hash:$salt format

### CODE REVIEW

**Comment by GrepItAll** [comment]
I think these are the only relevant files that need updating. Hopefully I haven't missed anything!

File: helpers/hashimporter.rb [hunk 1]
```diff
@@ -194,6 +194,22 @@ def importHashSalt(hash, hashfile_id, type)
   updateHashfileHashes(@hash_id.id.to_i, 'null', hashfile_id)
 end
 
+def importUserHashSalt(hash, hashfile_id, type)
+  data = hash.split(':', 2)
```
Comment (GrepItAll) [inline helpers/hashimporter.rb]: Splits into two parts. Originally I split this without the '2' argument, so it split into three parts and then had to be concatenated back together again. This is neater.

**Comment by i128** [comment]
This is awesome! Thanks for the good work

**Comment by GrepItAll** [comment]
No problem, glad it's of use!

**Pull request merged**